### DISCUSSION TITLE
Kiali graph is showing that service is sending traffic to nodes in "unknown" namespace

### DISCUSSION THREAD

**Discussion by akankshyad** [kiali/kiali#5548] [General]
I can see a lot of connections to multiple "Unknown" nodes from our services in the Kiali graph and they are failing.

I have meshConfig.outboundTrafficPolicy.mode set as REGISTRY_ONLY.

Why they are getting connected and how to restrict them?

Versions Used:
kiali version : v1.50.0
Istio version : istio-1.14.1
![kiali](https://user-images.githubusercontent.com/20989076/194828407-29c53db8-5da5-45db-8b12-45b4ab5a6cde.PNG)

**Comment by jshaughn** [comment]
Is that traffic trying to go external to the mesh?   If so you will need to define Service Entries if your external traffic policy is REGISTRY_ONLY.  If you change to ALLOW_ANY it is less secure but would then allow external requests to exit the mesh (via PASSTHROUGH_CLUSTER).  This is not a problem with Kiali, Kiali is just representing the telemetry provided by Istio.

**Reply by akankshyad** [reply]
Thanks @jshaughn for the response. External traffic policy is set to REGISTRY_ONLY and I have created Service Entries for the endpoints external to the mesh to which connectivity is fine. But in addition to those, traffic is going to external endpoints which are not part of Service Entry. Is there any way to restrict those? Please let me know. Thanks in advance.

**Reply by jshaughn** [reply]
With REGISTRY_ONLY requests to external hosts will be blocked unless they are handled by a service entry.  Other requests will be sent to BlackHoleCluster (basically a terminating handler).  So, I'm not exactly sure what you are seeing, but if you believe traffic is exiting the mesh mistakenly then you may need to pursue that issue with the Istio folks directly.
